pkg/controller/provider/controller.go

```diff
@@ -59,6 +65,17 @@ var log = logging.WithName(Name)
 // Application settings.
 var Settings = &settings.Settings
 
+const (
+	ovaContainerName = "ova"
+	ovaPodPrefix     = "ova"
+	imageVar         = "OVA_PARSER_IMAGE"
```

Comment:
Please use more specific name instead of the general `imageVar`, e.g. `ovaImageVar`. But I understand this is out of place and still WIP.